conans/test/assets/sources.py

```diff
@@ -0,0 +1,76 @@
+from jinja2 import Template
+
+_function_cpp = r"""
+{% if name != "main" %}
+#include "{{name}}.h"
+{% endif %}
+
+#include <iostream>
+
+{% for it in includes -%}
+#include "{{it}}.h"
+{%- endfor %}
```

Comment:
```suggestion
{% if includes %}{% for it in includes -%}
#include "{{it}}.h"
{%- endfor %}{% endif %}
```

or using a `default` filter